tutorials/en/vectara-advanced-app-tutorial-showcase-the-creation-of-vectara-app-in-legal-or-customer-support-use-case.mdx

```diff
@@ -0,0 +1,671 @@
+---
+title: "Vectara Advanced App Tutorial: Showcase The Creation Of Vectara App In Legal Or Customer Support Use Case"
+description: "Dive into a step-by-step guide on creating an intuitive Vectara application tailored for legal and customer support scenarios. Discover the intricacies of integrating semantic search and user-friendly interfaces, and get a first-hand look at the future of efficient customer support."
+authorUsername: "sanchayt743"
+---
+
+## 1. Introduction to the Vectara Ecosystem: 🌌
+
+Welcome to the realm of Vectara, a platform where search transcends to new horizons powered by Generative AI. Here, I will unravel the essence of the Vectara ecosystem, its fundamental workings, and provide a visual journey through Vectara's official materials to enhance your comprehension.
+
+### 🚀 Overview of Vectara Ecosystem
```

Comment:
Same comments as with the other tutorial (https://github.com/lablab-ai/community-content/pull/401/files) - not enough focus on GenAI and too much on "search". It's important to explain that Vectara can be used for semantic search but also for RAG applications and GenAI in this intro paragraph.

Reply:
Okay , will emphasise that point too